apollo-backend/src/main/java/io/logz/apollo/common/MDCLogging.java

```diff
@@ -0,0 +1,21 @@
+package io.logz.apollo.common;
+
+import org.slf4j.MDC;
+
+import java.util.Map;
+
+public class MDCLogging {
```

Comment:
Maybe MDCWrapper?

Reply:
Also thought about that 🙂 
It might be easier to find for people familiar with gaia-full.

Reply:
Never liked that name :)

Reply:
But looks like you're keen on the implementation :)

Reply:
Since they have lots of lines in the Apollo which only set and not clear, I figured I'll help @matvey-mtn with having that already, before he fixes all the other stuff 😉